components/openthread.rst

```diff
@@ -5,21 +5,20 @@ OpenThread Component
     :description: Instructions for setting up OpenThread component.
     :image: openthread.png
 
-`Thread <https://www.threadgroup.org>`__ is a low-power mesh networking standard for IoT devices. The low-power aspect is important for battery-powered smart home devices. However, it’s also low-bandwidth, making it ideal for applications that don’t send a lot of data, like switches or motion sensors.
+`Thread <https://www.threadgroup.org>`__ is a low-power, mesh-networking standard for IoT devices. The low-power aspect is important for battery-powered devices. It is also low-bandwidth, making it ideal for applications that do not send a lot of data, such as switches or motion sensors.
```

Comment:
You've changed "it's" to "it is" here, but in other places have replaced separate words with contractions.